<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Класс ЧС</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#ffff7f
}
QPushButton{
background-color:#55aaff;
border-style:outset;
padding:2px;
border-width:5px;
border-radius:7px;
border-color:#55aaff;
}
</string>
  </property>
  <widget class="QTableView" name="tVClass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDialog{background-color:rgb(255, 255, 127);}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>132</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="BtnAddClass">
      <property name="toolTip">
       <string>Добавить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/add-file.png</normaloff>../../../Downloads/add-file.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelClass">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Удалить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/delete.png</normaloff>../../../Downloads/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSaveClass">
      <property name="toolTip">
       <string>Сохранить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/diskette.png</normaloff>../../../Downloads/diskette.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tVVid">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>391</width>
     <height>431</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
  </widget>
  <widget class="QTableView" name="tVGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>132</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="BtnAddGroup">
      <property name="toolTip">
       <string>Добавить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/add-file.png</normaloff>../../../Downloads/add-file.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelGroup">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Удалить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/delete.png</normaloff>../../../Downloads/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSaveGroup">
      <property name="toolTip">
       <string>Сохранить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/diskette.png</normaloff>../../../Downloads/diskette.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>132</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="BtnAddVid">
      <property name="toolTip">
       <string>Добавить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/add-file.png</normaloff>../../../Downloads/add-file.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelVid">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Удалить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/delete.png</normaloff>../../../Downloads/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSaveVid">
      <property name="toolTip">
       <string>Сохранить</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/diskette.png</normaloff>../../../Downloads/diskette.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tVDirectory">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>541</width>
     <height>431</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>440</y>
     <width>198</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="BtnDelDirect">
        <property name="toolTip">
         <string>Удалить</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/delete.png</normaloff>../../../Downloads/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnSaveDirect">
        <property name="toolTip">
         <string>Сохранить</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/diskette.png</normaloff>../../../Downloads/diskette.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnAddDirect">
        <property name="toolTip">
         <string>Добавить</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/add-file.png</normaloff>../../../Downloads/add-file.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnRefreshDirect">
        <property name="toolTip">
         <string>Сохранить</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/refresh.png</normaloff>../../../Downloads/refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancelDirect">
        <property name="toolTip">
         <string>Сохранить</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/multiply.png</normaloff>../../../Downloads/multiply.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
